packages/react/src/components/List/List.tsx

```diff
@@ -291,7 +291,7 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
           }
         }
 
-        this._scrollElement.scrollTop = scrollTop;
+        this._scrollElement!.scrollTop = scrollTop;
```

Comment:
This should maybe be:

```suggestion
        if (this._scrollElement) {
            this._scrollElement.scrollTop = scrollTop;
        }
```